.gitpod.yml

```diff
@@ -0,0 +1,8 @@
+tasks:
+  - init: nvm install; nvm use; npm ci; npm run lerna-ci; npm run init-environment
```

Comment:
Same here, `node@14.x` comes with `npm@6.x` if I'm not mistaken but we're targeting `npm@8.x` which follows the `@next-8` on the npm registry.

Reply:
Yes node@14 does include npm@6 so I would expect that to be the norm until people move to node@16?

Reply:
Ok done.

Maybe we can have a helper make file like this?

```Makefile
dev:
	  nvm install
	  nvm use
	  npm i -g npm@next-8
	  npm ci
	  npm run lerna-ci
	  npm run init-environment
```